.github/workflows/ci.yml

```diff
@@ -0,0 +1,18 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [10.15.3]
```

Comment:
Can we raise the node version to 12?

Reply:
I added the setting of version 12, so it runs on both versions.

If you only need version 12, can you let me know?